pyriemann_qiskit/alias.py

```diff
@@ -0,0 +1,5 @@
+from pyriemann.estimation import ERPCovariances
+from pyriemann.classification import MDM
+from sklearn.pipeline import make_pipeline
+
+ERPCov_MDM = make_pipeline(ERPCovariances(estimator="lwf"), MDM())
```

Comment:
IMO, you should not create a new module only for a two-step pipeline.

You should follow the example of `SSVEP` folder in pyriemann examples, having a subfolder `helpers`,
to put all functions that can be re-use by other examples.

Reply:
Ok, it makes sense. But if these pipelines are used by different folders than `MI`, I am not sure how to resolve the file dependency then. It works for pyRiemann because there is no relative import. How do we proceed here?

Reply:
Since the pipeline `ERPCov_MDM` is written in one line, you can copy it in each example outside of `ERP` folder.